mllib/src/main/scala/org/apache/spark/mllib/clustering/KMeans.scala

```diff
@@ -109,35 +108,20 @@ class KMeans private (
   }
 
   /**
-   * :: Experimental ::
-   * Number of runs of the algorithm to execute in parallel.
+   * This function has no effect since Spark 2.0.0.
    */
   @Since("1.4.0")
-  @deprecated("Support for runs is deprecated. This param will have no effect in 2.0.0.", "1.6.0")
```

Comment:
```@deprecated``` means it take effect but not recommend to use, but in this PR we will make it no effect.